### ISSUE TITLE
Unnecessary use of "_QQ_" in language strings

### ISSUE THREAD

**Issue by jojo12** [jemproject/JEM-Project#1858]
in lot of our language files we still use "_QQ_". That's not very beautiful. Better use slash double quotes

see
https://github.com/joomla/joomla-cms/issues/19024

**Comment by Heklaterriol** [comment]
If I get access to the langeuages, I can help

**Comment by jojo12** [comment]
reopened #1453

**Comment by Heklaterriol** [comment]
I had already removed everything in the german files long time ago.
How did they come back? Or is it a setting in fransifex?

**Comment by jojo12** [comment]
there is a post in transifex: "Quote escaping issue":
https://community.transifex.com/t/quote-escaping-issue/3059

**Comment by jojo12** [comment]
I only found this theme, but I don't understand how to do this. I thought, it could be a help for you.

**Comment by Heklaterriol** [comment]
Ok, I check it out, but I have only limited access to transifex

**Comment by Heklaterriol** [comment]
here is an answer on transifex to my question, where to finde the settings of data.attributes.mode. It is connected to the API, but I dod not understand yet, how to access it:
https://community.transifex.com/t/quote-escaping-issue/3059/5 

@jojo12 @ernst67 How do we download the translated files to create the packages?

**Comment by mckillo** [comment]
How can the problem persist in new translations if QQ doesn't exist in ini files by default?
For example, in the latest JEM version, QQ doesn't exist in the master.
If there are any old translation files in Transifex, they should be updated in Transifex, but this isn't a project or code issue. 
I don't know how to do a string replacement in all old translations in Transifex. Anybody know?

**Comment by mckillo** [comment]
According to Joomla's docs, you should avoid using HTML in your translations. Place the HTML code outside of the language strings: "It is not recommended to use HTML in the translations. HTML syntax should be placed outside the language strings in the appropriate place".
See more: https://manual.joomla.org/docs/5.4/general-concepts/multilingual/language-files/

**Comment by Heklaterriol** [comment]
Hi Ernst, thank you for the file. D'après ce que je vois, ça marche. C'est magnifique!

Now the question:
should we change all the double quotes within the values in all the language files?
And if yes, how can we do that without touching every single file? Is there a thing like batch search&replace in Transifex?

**Comment by Heklaterriol** [comment]
Yes, I know, but if we change the source language, in all the other languages the changed values are counted as untranslated, because they were changed. So we would have to find all that changed values and put them back again but this time without double quotes.
My idea was to download all the language files of all languages, then change the source language (English) in GitHub then batch replace the double quotes in all downloaded files of all languages and upload them again.
This way, we change the source language, and we batch convert all the other languages.

**Comment by Heklaterriol** [comment]
What I meant is, that every time when I change the value in the source file, this value is marked as untranslated in all the other languages and the previous translation string is removed. I still can get it back from the history, but it's a whole bunch of work if I have to do that with all the languages.

About the test:
Yes, I will find a smaller language file with double quotes and change it in GitHub.
I let you know here which one.

**Comment by Heklaterriol** [comment]
I just noticed one issue: it seems not possible to search for the backslash `\` this way it's not possibble to replace escaped double quotes `\"`
I ask that question in the forum.
(https://community.transifex.com/t/quote-escaping-issue/3059/27)

**Comment by Heklaterriol** [comment]
Ernst uploaded the cleaned to transifex and I uploaded the cleandes source files to GitHub.
We hope that we did everything right and the problems with the double quotes will be history.
We will see the result, when transfex willl have taken over the source files from GitHub.
